android/fakedapp/src/main/res/values/strings.xml

```diff
@@ -24,6 +24,9 @@
     <string name="label_sign_msg_x1">Sign msg x1</string>
     <string name="label_sign_msg_x3">x3</string>
     <string name="label_sign_msg_x20">x20</string>
+    <string name="label_txn_version">Transaction Version:</string>
+    <string name="label_txn_version_legacy">Legacy</string>
```

Comment:
NIT: in the convention used in this test app (which is admittedly entirely arbitrary), `label_` is a prefix for widget labels, and `string_` is ususally used for, e.g., the contents of widgets.